response_operations_ui/controllers/party_controller.py

```diff
@@ -61,6 +61,8 @@ def get_respondent_by_party_id(respondent_party_id):
         response.raise_for_status()
     except requests.exceptions.HTTPError:
         log_level = logger.warning if response.status_code in (400, 404) else logger.exception
+        if response.status_code == 404:
+            return ''
```

Comment:
I think this would be better returning an empty json object rather than a string to keep the return value consistent